### DISCUSSION TITLE
Home Position - How to make Z Home Up?

### DISCUSSION THREAD

**Discussion by garyzz** [synman/Octoprint-Bettergrblsupport#89] [Q&A]
When I home my Z is set to LOW limit switch.

What do I need to configure to ensure that when I HOME machine Z is in the UP position?

**Comment by garyzz** [comment]
It turns out the default G23 code is set to 7 which I am learning may be Z down position.  I changed this to 3 and now my Z is homing in an UP position.  Hope this helps someone.  And I LOVE this plugin.  Amazing!

**Comment by synman** [comment]
yup... I  try really hard to stick to the foundation / boundaries defined with GRBL itself.  The vast majority of the plugin's behaviors are based on your GRBL settings.  Glad you got it figured out.

I just finished the mechanical / electronics build for my new Queen Bee Pro so expect to see some activity for the plugin in the coming weeks.

**Comment by garyzz** [comment]
Thank you.  Wow that Queen Bee Pro looks sweet!  I am just getting my feet wet with [this](https://www.amazon.com/ANNOYTOOLS-Switches-Emergency-Aluminum-3018PLUS/dp/B097DBQV9K/ref=sr_1_1?crid=11LH4XMZZ0QDZ&keywords=annoytools+cnc&qid=1667612549&qu=eyJxc2MiOiIwLjk3IiwicXNhIjoiMC4wMCIsInFzcCI6IjAuMDAifQ%3D%3D&sprefix=annoytools+cn%2Caps%2C137&sr=8-1&ufe=app_do%3Aamzn1.fos.ac2169a1-b668-44b9-8bd0-5ec63b24bcb5).
